modules/ROOT/pages/appendix/version-5.adoc

```diff
@@ -0,0 +1,407 @@
+:description: This section lists all Bolt message state transitions available in version 5.1+
+
+= Bolt message state transitions in version 5.1+
+
+This section lists all Bolt message state transitions available in version 5.1 +.
+
+[cols="20,15,20,30,30",options="header"]
+|===
+| State
+| Request message
+| Triggers signal
+| Server response summary message
+| New state
+
+| `NEGOTIATION`
+| `HELLO`
+|
+| `SUCCESS {}`
+| `AUTHENTICATION`
+
+| `NEGOTIATION`
+| `HELLO`
+|
+| `FAILURE {}`
+| `DEFUNCT`
+
+| `AUTHENTICATION`
+| `LOGON`
+|
+| `FAILURE {}`
+| `DEFUNCT`
+
+| `AUTHENTICATION`
+| `LOGON`
+|
+| `SUCCESS {}`
+| `READY`
+
+|
+|
+|
+|
+|
+
+| `READY`
+| `LOGOFF`
+|
+| `FAILURE {}`
+| `FAILED`
```

Comment:
Tangent: when would `LOGOFF` fail? I kinda didn't expect it could.

Reply:
It should never, as it stands currently, but if something happens that is unexpected it will endup in failed state as its part of ready state which is a fail safe state.